docs/monitoring.md

```diff
@@ -479,11 +479,17 @@ can be identified by their `[attempt-id]`. In the API listed below, when running
     <td><code>/applications/[app-id]/stages/[stage-id]</code></td>
     <td>
       A list of all attempts for the given stage.
+      <br><code>?details=true</code> list all attempts with stage detail data.
```

Comment:
<br><code>?details=true</code> lists all attempts with task data for the given stage.

Reply:
DOne